.gitignore

```diff
@@ -23,3 +23,10 @@ override.tf.json
 
 # Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
 # example: *tfplan*
+
+# Exclude the top level config file
+config.tfvars
+
+# Exclude the terraform state in the public template repo
```

Comment:
Yeah was torn with this. Yeah perhaps we do gitignore it just so we don't accidentally check any state in but we need to have an instruction in the readme that says ideally remove the gitignore as you should check in state blah blah blah

Reply:
why have an instruction when you could have [a script](https://github.com/UCLH-Foundry/Azure-Bootstrap/blob/t-young31/vmss_and_cis/scripts/init.py).. mostly for fools like me who don't read the docs

Reply:
Have seen your script further down - filled me with dread